tests/selenium_tests/test_new_excerpt.py

```diff
@@ -26,10 +26,13 @@ def test_new_excerpt(base_url, login, file_name, file_format, selenium, reload_u
     create.send_keys(Keys.RETURN)
 
     # wait until download link appears
-    selenium.find_element_by_link_text('↻ Reload')
-    element = reload_until_condition(selenium.find_element_by_class_name, "form-control")
+    selenium.find_element_by_xpath("/html/body/div/div/div[2]/div[2]/div[1]/div/div[1]/h3")
+    first_a = "/html/body/div/div/div[2]/div[2]/div[1]/div/div[2]/div[1]/div[1]/div[2]/div/div[1]/p/a"
```

Comment:
`find_element_by_link_text` was easier to read and matches more closely what a user would do. (They usually don't care about the DOM as long as the page looks good and they can find what they're looking for.) What's the reason to switch to `find_element_by_xpath`?

I guess it isn't arbitrary that these two XPaths have the common prefix `/html/body/div/div/div[2]/div[2]/div[1]/div/`. Should that prefix be extracted to a constant or local variable, so that it's name can give the reader some hint on what we're looking for in the DOM here?